frontend/simple/js/state.js

```diff
@@ -17,7 +16,15 @@ import debounce from 'lodash/debounce'
 Vue.use(Vuex)
 var store // this is set and made the default export at the bottom of the file.
           // we have it declared here to make it accessible in mutations
-
+// Used by the getters and mutations to find the mailbox contracts
+function getMailbox (state) {
```

Comment:
This function is not descriptive of what it does. `getMailbox` sounds like it returns a mailbox, but it doesn't, it returns the *contractId* of a mailbox. So it should say that. It's also a good idea to add a simple annotation to functions when this is the case, because that will make it clear to readers (and future-you) that it's returning a `string`, not a contract.

NOTE: If it doesn't make sense to return a contract (as that seems to be the case here), then it should read `getMailboxState`, or `mailboxStateForUser` [per comment below]).